src/main/java/com/onwelo/tinvalidator/validator/SEValidator.java

```diff
@@ -0,0 +1,31 @@
+package com.onwelo.tinvalidator.validator;
+
+import java.util.regex.Pattern;
+
+public class SEValidator implements Validator {
+
+    private static final Pattern PATTERN = Pattern.compile("[0-9]{12}");
+
+    public Pattern getPattern() {
+        return PATTERN;
+    }
+
+    public boolean computeControlSum(String tin) {
+        int r1 = Integer.parseInt(tin.substring(11));
+        if (r1 == 0 || r1 > 94) {
```

Comment:
You can consider using an assertion instead of if clause for the sake of code consistency